docs/developers/developer-reference/mandos-tests-reference.md

```diff
@@ -272,6 +272,54 @@ This step simulates a transaction to an existing smart contract. Fields:
     - `data` - same as the topics, but this is not indexed, cannot perform searches on data. Can be of any length (or sometimes empty).
   - `gas` - here the consumed gas can be checked. To ignore this check, set to `“*”`
   - `refund` - some operations, like freeing up storage actually give ERD back to the caller. To ignore this check, set to `“*”`
+  
+## **Step type: `scQuery`**
+
+```
+{
+    "steps": [
+        {
+            "step": "scQuery",
+            "txId": "tx-name-or-id",
+            "comment": "just an example",
+            "tx": {
+                "to": "0x1000000000000000000000000000000000000000000000000000000000000000",
+                "function": "someFunctionName",
+                "arguments": [
+                    "0x1234123400000000000000000000000000000000000000000000000000000004",
+                    "0x00",
+                    "",
+                    "``a message (as bytes)"
+                ],
+            },
+            "expect": {
+                "out": [
+                    "5",
+                    "*"
+                ],
+                "status": "",
+                "gas": "*",
+                "refund": "*"
+            }
+        }
+    ]
+}
+```
+
+This step simulates a query to an existing smart contract. Fields:
+
+- `txId` (optional) - it shows up in the error messages, to help the developer find a transaction that produced a wrong result or that failed. It is also used to generate mock tx hashes.
+- `comment` (optional) - developers can provide comments or a description of the transaction. Does not influence execution.
+- `tx` - specifies the details of the transaction.
+  - `to` - account must exist in the blockchain mock and must be a smart contract
+  - `function` - function name to call in the contract
+  - `arguments` - a list of the arguments to provide to the SC function
+- `expect` (optional) - each transaction produces a receipt whose hash ends up on the blockchain. The contents of the receipt can be checked here.
+  - `out` - functions can return any number of results. This is an ordered list of these results.
+  - `status` - indicates whether execution completed successfully or not. Status 0 means success. All errors occurring in the contract will yield status 4 (“user error”).
+  - `message` (optional) - in case of error, the contract can also provide an error message. This is where this message can be checked, to make sure that the correct error occurred. It will be empty in case of success.
+  - `gas` - here the consumed gas can be checked. To ignore this check, set to `“*”`
+  - `refund` - some operations, like freeing up storage actually give ERD back to the caller. To ignore this check, set to `“*”`
```

Comment:
EGLD? or simply `tokens`

Reply:
Ah, copy paste error since I just copied from scCall and didn't double check the infos there. Will fix for both! :)

Reply:
I settled for eGLD since it's more clear to the users and I don't think other tokens will be freed :)